packaging/docker/Dockerfile

```diff
@@ -58,9 +58,11 @@ COPY --from=builder /app /
 # Configure system
 ARG NETDATA_UID=201
 ARG NETDATA_GID=201
+ENV DOCKER_GRP netdata
+ENV DOCKER_USR netdata
 RUN \
     # provide judy installation to base image
-    apk add make alpine-sdk && \
+    apk add make alpine-sdk shadow && \
```

Comment:
I was wondering, why do we need shadow package?

Reply:
That's to add the `usermod` command for adding the `netdata` user to the `docker` group on run.

https://github.com/netdata/netdata/pull/6441/files#diff-8ab2a2dd9f9a25a877e05f48b46a7ee3R20